examples/chip-tool/BUILD.gn

```diff
@@ -24,14 +24,24 @@ declare_args() {
   config_use_separate_eventloop = true
 }
 
-executable("chip-tool") {
+config("config") {
+  include_dirs = [
+    ".",
+    "${chip_root}/zzz_generated/chip-tool",
+  ]
+
+  defines = [ "CONFIG_USE_SEPARATE_EVENTLOOP=${config_use_separate_eventloop}" ]
+}
```

Comment:
What is the reason for this new target?

Reply:
To share both defines and includes with static library and executable targets